### PR TITLE
Cross-browser issue fix, new spinner, pictures in auto-complete

### PR DESCRIPTION
**Pull request by jsilvestre**
It's pretty straight-forward!

For the contact picture, I copied what is done in Files+cozy-clearance.

### CODE REVIEW

**Comment by jsilvestre** [comment]
I fixed what you mentioned, thank for the review.

**Comment by jsilvestre** [comment]
Nvm it doesn't work...let me fix it.

**Comment by jsilvestre** [comment]
What's wrong with me, it's fixed (typo...).

File: server/controllers/index.coffee [hunk 1]
```diff
@@ -52,3 +52,10 @@ module.exports.index = (req, res) ->
                 window.webDavAccount = #{JSON.stringify webDavAccount};
                 window.timezone = #{JSON.stringify timezone};
             """
+
+
+module.exports.contactPicture = (req, res, next) ->
```
Comment (frankrousseau) [inline server/controllers/index.coffee]: I think there is an helper in CozyDB for thak kind of controller.

**Comment by jsilvestre** [comment]
I changed the code to use SimpleController feature for contacts.

**Pull request merged**